modules/managing-memcached-operator-using-olm.adoc

```diff
@@ -46,26 +50,48 @@ for more information on manually defining a manifest file.
 
 . *Deploy the Operator.*
 
-.. Deploy an Operator by applying the Operator’s manifest to the desired namespace
-in the cluster:
+.. Create an OperatorGroup that specifies the namespaces that the Operator will
+target. Create the following OperatorGroup in the namespace where you will
+create the CSV. In this example, the `default` namespace is used:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha2
+kind: OperatorGroup
+metadata:
+  name: memcached-operator-group
+  namespace: default
+  spec:
+    targetNamespaces:
+    - default
+----
+
+.. Apply the Operator's CSV manifest to the specified namespace in the cluster:
 +
 ----
 $ curl -Lo memcachedoperator.0.0.1.csv.yaml https://raw.githubusercontent.com/operator-framework/getting-started/master/memcachedoperator.0.0.1.csv.yaml
 $ oc apply -f memcachedoperator.0.0.1.csv.yaml
 $ oc get ClusterServiceVersion memcachedoperator.v0.0.1 -o json | jq '.status'
 ----
++
+When you apply this manifest, the cluster does not immediately update because
+it does not yet meet the requirements specified in the manifest.
 
-.. After applying this manifest, nothing has happened yet, because the cluster does
-not meet the requirements specified in our manifest. Create the RBAC rules and
-`CustomResourceDefinition` for the Memcached type managed by the Operator:
+.. Create the role, role binding, and service account to grant resource
+permissions to the Operator, and the Custom Resource Definition (CRD) to create
+the Memcached type that the Operator manages:
 +
 ----
-$ oc apply -f deploy/rbac.yaml
-$ oc apply -f deploy/crd.yaml
+$ oc create -f deploy/crds/cache_v1alpha1_memcached_crd.yaml
+$ oc create -f deploy/service_account.yaml
+$ oc create -f deploy/role.yaml
+$ oc create -f deploy/role_binding.yaml
 ----
 +
-NOTE: The `rbac.yaml` and `crd.yaml` files were generated into the `deploy/`
-directory by the Operator SDK when you built the Memcached Operator.
+[NOTE]
+====
+These files were generated into the `deploy/` directory by the Operator SDK when you built the Memcached Operator.
+====
 +
 Because the OLM creates Operators in a particular namespace when a manifest is
 applied, administrators can leverage the native Kubernetes RBAC permission model
```

Comment:
I cannot leave comments on line 102, so comments here: "The Memcached Operator is now running in the `memcached` namespace." should running in `default` namespace, not `memcached`.